source/features/softwrap-code.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+Soft-wrap code inside codeblocks
+*/
+import select from 'select-dom';
+import features from '../libs/features';
+
+const tabSize = 4;
+
+async function init(): Promise<void> {
+	// Classes:
+	// `.diff-table` - Split and unified diffs
+	// `.d-table` - Suggested changes
+	// `.js-file-line-container` - Embedded code blocks
+	const tables = select.all('.diff-table, .d-table, .js-file-line-container');
```

Comment:
You can skip this and just select `.blob-code-inner`

Reply:
Need this, as we target the classes added to these tables to change the layout of tables a bit.